beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -1015,6 +1025,10 @@ impl<T: BeaconChainTypes> Worker<T> {
                 );
             }
         };
+
```

Comment:
There's a `trace!` just above this that tries to log the invalid block as SSZ. I think we may as well remove that (I don't think anybody likes running with trace logs).

Reply:
@AgeManning IIRC you've used this in the past. Are you OK if we remove it?

Reply:
The SSZ will still be accessible on the filesystem, but you'd have to run with the flag provided to get it.